webapp/src/components/soundEffectEditor/soundUtil.ts

```diff
@@ -289,4 +289,19 @@ export function getMixerGallerySounds(): SoundGalleryItem[] {
     ];
 
     return res;
+}
+
+function readDalConstant(name: string, defaultValue: number) {
+    const file = pxt.appTarget.bundledpkgs?.["core"]?.["dal.d.ts"];
+
+    if (!file) return defaultValue;
+
+    const regEx = new RegExp(`${name}\\s*=\\s*(\\d+),`);
+
+    const match = regEx.exec(file);
+
+    if (match) {
+        return parseInt(match[1]);
```

Comment:
Should we protect against non-numeric strings here and return defaultValue?

Reply:
Oh nvm, I guess regex does this already

Reply:
yeah, also these are always numbers